### PR TITLE
Zoomv1.0.3

### PR DESCRIPTION
**Pull request by roopkumark**
Update to tap-code to pull Zoom Phone Recordings

### CODE REVIEW

File: tap_zoom/sync.py [hunk 1]
```diff
@@ -202,7 +203,14 @@ def sync_endpoint(client,
...
         if data is None:
             return
-
+        
+        # Skip parsing records, if an API call returns 0 'total_records'  
+        if 'record_count_key' in endpoint:
+            record_count = records = data[endpoint['record_count_key']]
+            
+            if record_count == 0:
+                return
```
Comment (ecomohr) [inline tap_zoom/sync.py:212]: nice!

**Review by ecomohr — APPROVED** [review]
lgtm

File: .gitignore [hunk 2]
```diff
@@ -121,3 +121,5 @@ singer-check-tap-data
 state.json
 catalog.json
 
+archive/phone_recordings.json
+config_hcp.json
```
Comment (vorpal56) [inline .gitignore]: this is not relevant here. can you exclude this?

**Review by vorpal56 — APPROVED** [review]
lgtm. didn't get to testing it though

**Pull request merged**